File: TheComfortZone.DTO/Order/OrderResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TheComfortZone.DTO.Order
{
    public class OrderResponse
    {
        public int OrderId { get; set; }
        public DateTime? OrderDate { get; set; }
        public string Status { get; set; }
        public int OrderNumber { get; set; }
        public int? Noip { get; set; }
        public float? TotalPrice { get; set; }
        public bool? UsedDiscountCoupon { get; set; }
        public string Customer { get; set; }
    }
}

File: the_comfort_zone_mobile/lib/model/order/order_response.dart
import 'package:json_annotation/json_annotation.dart';
part 'order_response.g.dart';

@JsonSerializable()
class OrderResponse {
  int? orderId;
  DateTime? orderDate;
  String? status;
  int? orderNumber;
  int? noip;
  double? totalPrice;

  OrderResponse(){}

  factory OrderResponse.fromJson(Map<String, dynamic> json) => _$OrderResponseFromJson(json);
  /// Connect the generated [_$OrderResponseToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$OrderResponseToJson(this);
}
